LEGO1/lego/legoomni/include/act2actor.h

```diff
@@ -2,6 +2,7 @@
 #define ACT2ACTOR_H
 
 #include "legoanimactor.h"
+#include "legopathedgecontainer.h"
```

Comment:
I don't think we need this include in the header? Let's move it to the compilation unit to avoid an extra dependency

Reply:
I must have added it to the wrong file by accident, thanks